src/pool/srv_target.c

```diff
@@ -864,11 +867,10 @@ ds_pool_stop(uuid_t uuid)
 
 	ds_pool_failed_remove(uuid);
 
-	pool = ds_pool_lookup(uuid);
+	ds_pool_lookup(uuid, &pool);
 	if (pool == NULL)
 		return;
```

Comment:
[Not requesting changes] From a code consistency point of view, I find it a little annoying to come across several different patterns for checking the result of a ds_pool_lookup call:

- Ignore the return value, and check pool: Here.
- Take the return value, check pool, and use rc: New line 1870, pool/srv_util.c new line 869, rebuild/srv.c new line 1290.
- Take the return value, check pool, but set rc to 0 without using its original value: rebuild/srv.c new line 489.

If we only assign to output parameter when returning zero, things would likely be simpler for code readers.

Reply:
The function might need better comments here indeed.

Reply:
The unchecked return value of ds_pool_lookup() here was reported in a Coverity scan, CID 979267